HdmiCecSink/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_include_directories(${MODULE_NAME} PRIVATE ${CEC_INCLUDE_DIRS})
 target_include_directories(${MODULE_NAME} PRIVATE ${DS_INCLUDE_DIRS})
 
 target_link_libraries(${MODULE_NAME} PUBLIC ${NAMESPACE}Plugins::${NAMESPACE}Plugins ${IARMBUS_LIBRARIES} ${CEC_LIBRARIES} ${DS_LIBRARIES} )
+target_compile_options(${MODULE_NAME} PRIVATE -Wall -Werror -Wno-error=deprecated)
```

Comment:
please remove flags from the project files

Reply:
Done